lighty-modules/lighty-gnmi/lighty-gnmi-sb/src/main/java/io/lighty/gnmi/southbound/mountpoint/codecs/YangInstanceNormToGnmiUpdateCodec.java

```diff
@@ -37,12 +39,16 @@ public class YangInstanceNormToGnmiUpdateCodec implements
     private final YangInstanceIdentifierToPathCodec toPathCodec;
     private final SchemaContextProvider schemaContextProvider;
     private final Gson gson;
+    private final boolean sendPrimitiveValAsJsonIetfVal; // needed for BroadCom Sonic 3.1.2
+
 
     public YangInstanceNormToGnmiUpdateCodec(final SchemaContextProvider schemaContextProvider,
-            final YangInstanceIdentifierToPathCodec toPathCodec, final Gson gson) {
+            final YangInstanceIdentifierToPathCodec toPathCodec, final Gson gson,
+                                             final boolean sendPrimitiveValAsJsonIetfVal) {
```

Comment:
Please remove all changes in this class.
1) Property `sendPrimitiveValAsJsonIetfVal` is set from `use-model-name-prefix` which is not related.
2) JSON_IETF value should be set as it is according to gNMI specification:
https://github.com/openconfig/reference/blob/master/rpc/gnmi/gnmi-specification.md#231-json-and-json_ietf

So GET/SET JSON_IETF value should look like:
```
For /a/b[name=b1]/c/d:

update: <
  path: <
    elem: <
      name: "a"
    >
    elem: <
      name: "b"
      key: <
        key: "name"
        value: "b1"
      >
    >
    elem: <
      name: "c"
    >
    elem: <
      name: "d"
    >
  >
  val: <
    json_val: "AStringValue"
  >
>
```

Reply:
No response from author.